### PR TITLE
UserAvatar component

### PR DESCRIPTION
**Pull request by RizzoMichael**

### CODE REVIEW

**Comment by AlixH** [comment]
I don't see the images in transactions history
And I have a superposition of initials and defautl image on the sidebar 
Screenshot 2021-05-31 at 14 37 05

**Comment by RizzoMichael** [comment]
> I don't see the images in transactions history
> And I have a superposition of initials and defautl image on the sidebar (iPhone only)
> Screenshot 2021-05-31 at 14 37 05

This problem only exists for the dark version.
I'll fix it.

**Comment by AlixH** [comment]
Screenshot 2021-05-31 at 14 27 02
If you can, trye to have the same size for the circles

**Comment by LucasBrazi06** [comment]
> 
> If you can, try to have the same size for the circles

Please ANONYMIZE the sensitive user's data !!!

**Comment by AlixH** [comment]
> > If you can, try to have the same size for the circles
> 
> Please ANONYMIZE the sensitive user's data !!!

Screenshot 2021-05-31 at 14 27 02
I you can, try to have the same size for the circles

**Review by AlixH — APPROVED** [review]

File: src/components/car/CarComponent.tsx [hunk 1]
```diff
@@ -71,7 +71,7 @@ export default class CarComponent extends React.Component<Props, State> {
               }}>
               <View style={style.userContainer}>
                 <View style={[style.avatarContainer]}>
-                  <UserAvatar small={true} user={defaultCarUser?.user} navigation={navigation} />
+                  <UserAvatar size={35} user={defaultCarUser?.user} navigation={navigation} />
```
Comment (LucasBrazi06) [inline src/components/car/CarComponent.tsx:74]: Will the size scale on iPad?

File: src/components/user/avatar/UserAvatar.tsx [hunk 2]
```diff
@@ -39,8 +40,18 @@ export default class UserAvatar extends React.Component<Props, State> {
...
       user.image = await this.getUserImage(user.id as string);
+      this.setState({ user });
+    }
+  }
+
+  public async componentDidUpdate() {
+    const { user } = this.props;
+    if (user) {
+      user.image = await this.getUserImage(user.id as string);
+      if (JSON.stringify(this.state.user) !== JSON.stringify(user)) {
```
Comment (LucasBrazi06) [inline src/components/user/avatar/UserAvatar.tsx:51]: Why not comparing user ID?

**Review by LucasBrazi06 — APPROVED** [review]

**Pull request merged**